x-pack/plugins/index_management/public/services/api.js

```diff
@@ -32,6 +66,9 @@ export async function closeIndices(indices) {
     indices
   };
   const response = await httpClient.post(`${apiPrefix}/indices/close`, body);
```

Comment:
For consistency, I think it would have been better:

```js
const actionType = indices.length > 1 ? UA_INDEX_CLOSE_MANY : UA_INDEX_CLOSE;
const request = httpClient.post(`${apiPrefix}/indices/close`, body);
const response = await trackUserRequest(request, actionType);
...
```

You know, I'm all about consistency 😊 Aside, the `trackUserRequest()` (that will come as a public method from the userAction plugin in the new platform) being an abstraction, we could easily extend it's behaviour in in a single place whenever needed.

Reply:
We discussed this, found some ways to make `trackUserRequest` easier to understand, and agreed it's OK to have different implementations from app-to-app for now, with an eye towards unifying them in the future once the New Platform is in a state that supports this effort.